include/mips/exc.h

```diff
@@ -56,7 +72,10 @@
   SAVE_REG(t0, SR, reg);                                                       \
   SAVE_REG(t1, CAUSE, reg);                                                    \
   SAVE_REG(t2, BADVADDR, reg);                                                 \
-  SAVE_REG(t3, PC, reg)
+  /* Saving value of user-space PC just before syscall. */                     \
+  SAVE_REG(t3, PC, reg, NO_CFI);                                               \
+  .cfi_return_column t3;                                                       \
```

Comment:
What does `.cfi_return_column` do ?

Reply:
According to doc: 
> Change return column register, i.e. the return address is either directly in register or can be accessed by rules for register.

I only use it to later insert `.cfi_rel_offset t3, EXC_PC`. To my best knowledge this is the only way to specify return address to instruction just before entering kernel-space.

Reply:
Without this we miss last frame before entering kernel-space. I.e. `write` which calls syscall.